tests/installation/add_maintenance_repos.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
```

Comment:
we always use a folder to distinguish libyui for non-libyui (not written anywhere, sorry, you didn't have to know that). I guess it should belong to `tests/installation/add_on_product`

Reply:
That's good to know, it may not be ideal but there's some logic to differentiate between libyui and non-libyui. Done!